dist/debian/rules

```diff
@@ -91,6 +97,7 @@ endif
 		--mandir=/usr/share/man
 
 override_dh_auto_build:
+	mkdir -p $(GOCACHE)
```

Comment:
Add an '@' before this so it isn't printed by default, just like the next line.

Reply:
I got it. I will update this line.

Reply:
I added '@'